tasks/request_from_archive/request_from_archive.py

```diff
@@ -556,30 +564,30 @@ def restore_object(
     s3_cli: BaseClient,
     key: str,
     days: int,
-    db_glacier_bucket_key: str,
+    db_archive_bucket_key: str,
     attempt: int,
     job_id: str,
-    recovery_type: str = "Standard",
```

Comment:
Since this is always called with the parameter present, the default was never used.

Reply:
Concur, I also feel this value should never be None and should always be set.